server/src/main/java/org/apache/iotdb/db/engine/cache/TimeSeriesMetadataCache.java

```diff
@@ -118,7 +118,9 @@ public TimeseriesMetadata get(
     if (!CACHE_ENABLE) {
       // bloom filter part
       TsFileSequenceReader reader = FileReaderManager.getInstance().get(key.filePath, true);
-      BloomFilter bloomFilter = reader.readBloomFilter();
+      BloomFilter bloomFilter =
+          BloomFilterCache.getInstance()
+              .get(new BloomFilterCache.BloomFilterCacheKey(key.filePath));
```

Comment:
If the `CACHE_ENABLE` is false, there is no need to read from `BloomFilterCache`, just use `reader.readBloomFilter()`.